js/abp-preprocessed.es6.js

```diff
@@ -44,6 +46,12 @@ function updateLists () {
             abp.parse(listData, easylists[list].parsed)
             easylists[list].loaded = true;
         });
+
+        load.loadExtensionFile({url: settings.getSetting("trackerWhitelist")}, function(listData, response) {
+            console.log("loaded tracker whitelist: " + listData);
+            abp.parse(listData, trackerWhitelist);
+
+        });
```

Comment:
A few things here:
- the console output shows this loading twice, not sure why. please fix as this could get slow with many rules.
- please don't output the list data into console as it will obscure other important init messages in overview
- please follow console output conventions used elsewhere: `Loaded: tracker whitelist`
- remove whitespace on line 53 and use single quotes on line 50

Reply:
ha. nice